test/DoctrineBatchUtilsTest/BatchProcessing/SimpleBatchIteratorAggregateTest.php

```diff
@@ -75,22 +75,25 @@ public function testIterationWithEmptySet(): void
     {
         $iterator = SimpleBatchIteratorAggregate::fromArrayResult([], $this->entityManager, 100);
 
-        $this->entityManager->expects(self::at(0))->method('beginTransaction');
-        $this->entityManager->expects(self::at(1))->method('flush');
-        $this->entityManager->expects(self::at(2))->method('clear');
-        $this->entityManager->expects(self::at(3))->method('commit');
```

Comment:
Similar to above: the execution order, due to the squishy/stateful nature of the ORM, is to be tested, therefore the `->at()` logic must be migrated to a different mechanism, rather than being removed